stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/dto/ChannelInfoDto.kt

```diff
@@ -16,14 +16,15 @@
 
 package io.getstream.chat.android.client.api2.model.dto
 
+import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
 
 @JsonClass(generateAdapter = true)
 internal data class ChannelInfoDto(
-    val cid: String?,
-    val id: String?,
-    val member_count: Int = 0,
-    val name: String?,
-    val type: String?,
-    val image: String?,
+    @Json(name = "cid") val cid: String?,
```

Comment:
We broke the convention on purpose while implementing Moshi. You can find Marton's explanation [here](https://github.com/GetStream/stream-chat-android/pull/1274#discussion_r564588533).
For me, DTOs classes sound like a good candidate for the exception

Reply:
Ummm, sounds good. I will add suppress annotation then to keep it